### ISSUE TITLE
Allow trailing commas in generic type list

### ISSUE THREAD

**Issue by ianobermiller** [facebook/flow#1210]
For consistency with the rest of ES6, it would be nice if Flow allowed trailing commas to appear in generic type lists, for example:

```js
class Foo extends Bar<
  Baz,
  Boz,
  string, // trailing comma, yay
> {
  // ...
}
```

**Comment by gabelevi** [comment]
Huh, I wonder why we didn't do that in the first place...

**Comment by samwgoldman** [comment]
There are probably more opportunities for trailing commas than just type parameter lists. `extends` lists and `mixins` at least.

**Comment by samwgoldman** [comment]
Fixed in #1470
